site/operational-guide/concepts.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Concepts
+menu_order: 10
+---
+This section describes some essential concepts with which you will
+need to be familiar before moving on to the example deployment
+scenarios.
+
+## Host
+
+For the purposes of this documentation we consider a host to be an
+installation of the Linux operating system which is running an
+instance of the Docker Engine. It may be executing directly on bare
+hardware or inside a virtual machine.
+
+## Peer
+
+A peer is a running instance of Weave Net, typically one per host.
+
+## Peer Name
+
+Peers in the weave network are identified by a 48-bit value formatted
+like an ethernet MAC address e.g. `01:23:45:67:89:ab`. This 'peer
+name' is used for various purposes:
+
+* Routing of packets between containers on the overlay network
+* Recording the origin peer of DNS entries
+* Recording ownership of IP address ranges
+
+Whilst it is desirable for the peer name to remain stable across
+restarts, it is essential that it is unique - if two or more peers
+share the same name chaos will ensue, including but not limited to
+double allocation of addresses and inability to route packets on the
+overlay network. Consequently when the router is launched on a host it
```

Comment:
Possibly `site/how-it-works/network-topology.md` which uses the term "peer name" a lot already (without explanation).